### PR TITLE
Added examples for policy commands

### PR DESCRIPTION
**Pull request by sagar-connect**
# Checklist

=========

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have checked for similar issues and haven't found anything relevant.
- [x] This is not a security issue (which should be reported here: https://circleci.com/security/)
- [x] I have read [Contribution Guidelines](https://github.com/CircleCI-Public/circleci-cli/blob/master/CONTRIBUTING.md).

## Changes

=======

- Added examples for policy commands

### CODE REVIEW

File: cmd/policy/policy.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func NewCommand(config *settings.Config, preRunE validator.Validator) *cobra.Com
 				return nil
 			},
 			Args:    cobra.ExactArgs(1),
-			Example: `policy push ./policy_bundle_dir_path --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f --context config`,
+			Example: `policy push ./config --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f`,
```
Comment (davidmdm) [inline cmd/policy/policy.go:86]: The folder to be pushed being called config is confusing. 

Policy push ./policies seems better.

There's the confusion between the context config and the folder config...

File: cmd/policy/policy.go [hunk 2]
```diff
@@ -117,7 +117,8 @@ func NewCommand(config *settings.Config, preRunE validator.Validator) *cobra.Com
 
 				return prettyJSONEncoder(cmd.OutOrStdout()).Encode(diff)
 			},
-			Args: cobra.ExactArgs(1),
+			Args:    cobra.ExactArgs(1),
+			Example: `policy diff ./config --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f`,
```
Comment (davidmdm) [inline cmd/policy/policy.go:121]: Same as above

File: cmd/policy/policy.go [hunk 3]
```diff
@@ -356,7 +358,8 @@ func NewCommand(config *settings.Config, preRunE validator.Validator) *cobra.Com
 
 				return nil
 			},
-			Args: cobra.ExactArgs(1),
+			Args:    cobra.ExactArgs(1),
+			Example: `policy eval ./config --input ./.circleci/config.yml`,
```
Comment (davidmdm) [inline cmd/policy/policy.go:362]: Same as above

**Review by davidmdm — APPROVED** [review]

File: cmd/policy/policy.go [hunk 4]
```diff
@@ -264,13 +265,13 @@ func NewCommand(config *settings.Config, preRunE validator.Validator) *cobra.Com
 
 		cmd := &cobra.Command{
 			Short: "make a decision",
-			Use:   "decide",
+			Use:   "decide [policy_file_or_dir_path]",
```
Comment (eddiewebb) [inline cmd/policy/policy.go:268]: 😄

**Review by eddiewebb — COMMENTED** [review]
Awesome, thanks @sagar-connect,

**Pull request merged**